README.md

```diff
@@ -120,6 +120,29 @@ npm install medusa-extender
 
 That's it, you are now ready to run your server :rocket:
 
+Note: If you want to run the server on a custom port update medusa-config.js as follows:
+
+```
+module.exports = {
```

Comment:
you could replace with 
```
module.exports = {
  /* ... */
  serverConfig:{
	  port: <PORT_NUMBER>
  }
  /* ... */
};
```

Reply:
you will also need to update the `documentations/01...` file and run npm run build:doc

Reply:
Thanks for the pointer! Updated accordingly